### DISCUSSION TITLE
Programmatically using goToTimecode to fully automate moving to a specific time in video

### DISCUSSION THREAD

**Discussion by VinayValsaraj-at** [mifi/lossless-cut#2086] [Q&A]
Hi! Curious if anyone has a solution for programmatically moving the cursor to a specified timestamp?

Using the `goToTimecode` keyboard shortcut programmatically through the API _works_, but requires user input on the application for what timestamp. Ideally this request could also intake a timestamp to move to.

Has anyone solved this issue? Or does this require an update to the repo to support this?

Thanks!

**Comment by mifi** [comment]
#2087
